src/components/images/imageLoader.js

```diff
@@ -11,6 +12,76 @@ import 'css!./style';
         fillImageElement(elem, source);
     }
 
+    // function destroyBlurhash(target) {
+    //     let canvas = target.getElementsByClassName('blurhash-canvas')[0];
+    //     target.removeChild(canvas);
+    //     target.classList.remove('blurhashed');
+    // }
+
+    function itemBlurhashing(entry) {
+        // This intersection ratio ensures that items that are near the borders are also blurhashed, alongside items that are outside the viewport
+        // if (entry.intersectionRation <= 0.025)
+        if (entry.target) {
+            let target = entry.target;
+            // We only keep max 80 items blurhashed in screen to save memory
+            // if (document.getElementsByClassName('blurhashed').length <= 80) {
+
+            //} else {
+                // destroyBlurhash(target);
+            //}
+            let blurhashstr = target.getAttribute('data-blurhash');
+            if (blurhash.isBlurhashValid(blurhashstr) && target.getElementsByClassName('blurhash-canvas').length === 0) {
+                console.log('Blurhashing item ' + target.parentElement.parentElement.parentElement.getAttribute('data-index') + ' with intersection ratio ' + entry.intersectionRatio);
+                // let width = target.offsetWidth;
+                // let height = target.offsetHeight;
+                let width = 18;
+                let height = 18;
+                if (width && height) {
```

Comment:
this `if` is not needed now because you define `width` and `height` yourself as constants... also width & height should probably follow image aspect ratio if that is known beforehand

Reply:
> also width & height should probably follow image aspect ratio if that is known beforehand

We should at least have the card shape at that point, so we have an estimation of it, at least.

Reply:
The estimation is not needed for anything anyway, because the canvas will always fill the size using CSS. I think it's safe to remove and follow @JustAMan suggestions.

Reply:
No, my comment was about making canvas have proper aspect ratio so it's scaled "more correctly". It's work regardless, but would look prettier if scaled uniformly.

Reply:
@JustAMan @MrTimscampi Everything I do here is based in https://blurhash-wasm.netlify.app/ implementation. They don't scale the canvas at all, they just use CSS for that. Blurhash implementation in their page is exactly the same.

Canvas and pixels must have the same width & height as well: otherwise the canvas won't be 100% covered (which is what we discussed on Matrix)